spec/views/hq/sponsors/_sponsorships_active_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'hq/sponsors/_sponsorships_active.html.haml', type: :view do
+  let(:orphan) {build_stubbed :orphan}
+  let(:sponsorships_active) {(1..3).map {build_stubbed :sponsorship, active: true, orphan: orphan}}
```

Comment:
You are potentially setting yourself up for trouble here - an orphan is only allowed to have 1 active sponsorship. If validations get triggered anywhere down the line, things will break for reasons that have nothing to do with this test. While that is unlikely in a view spec, I'd still prefer not to knowingly create invalid data.

Looking at the following tests, I don't think reusing a single orphan object buys anything. L4 can be simply deleted and reference to it from L5 removed.